build/bootstrap_mcs.sh

```diff
@@ -16,7 +16,7 @@ fi
 
 message "Building Mariadb Server from $color_yellow$MDB_SOURCE_PATH$color_normal"
 
-BUILD_TYPE_OPTIONS=("Debug" "RelWithDebInfo")
+BUILD_TYPE_OPTIONS=("Debug" "RelWithDebInfo" "Release")
```

Comment:
JFYI I am aware that Release build type is error-prone in case of MDB/MCS and it is never compiled or used thus this will be an unused option. Could you remove this?

Reply:
Will do.

Reply:
Removed it is.